app/controllers/application_controller.rb

```diff
@@ -1,5 +1,2 @@
 class ApplicationController < ActionController::Base
-  def current_user
-    User.first
```

Comment:
- [ ] In order to permit additional parameters (input fields) like `username` in the devise registration view, Kindly do that using a simple before action in your ApplicationController like that:

```ruby
  before_action :update_allowed_parameters, if: :devise_controller?

  protected

  def update_allowed_parameters
    devise_parameter_sanitizer.permit(:sign_up) { |u| u.permit(:name, :email, :password) }
    devise_parameter_sanitizer.permit(:account_update) do |u|
      u.permit(:name, :email, :password, :current_password)
    end
  end
```

Reply:
of course, I'm working on that